settings.gradle.kts

```diff
@@ -28,6 +28,22 @@ dependencyResolutionManagement {
     mavenCentral()
     mavenLocal()
   }
+
+  versionCatalogs {
+    val latestDepTest = gradle.startParameter.projectProperties["testLatestDeps"] == "true"
+    create("springBoot2") {
+      val springBoot2Version =
+        gradle.startParameter.projectProperties["springBoot2Version"] ?:
+        (if (latestDepTest) "2.7.18" else "2.4.2")
+      plugin("versions", "org.springframework.boot").version(springBoot2Version)
+    }
+    create("springBoot3") {
+      val springBoot3Version =
+        gradle.startParameter.projectProperties["springBoot3Version"] ?:
+        (if (latestDepTest) "3.3.0" else "3.0.0")
```

Comment:
any idea if we can tie the latest dep version (`3.3.0`) to the artifact somehow so that it will get auto-updated by renovate?

Reply:
have you tired using `+` as the version?